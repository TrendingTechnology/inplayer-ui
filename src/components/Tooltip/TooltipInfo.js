import styled from 'styled-components';
import { uiColors } from 'utils';

const TooltipInfo = styled.div`
  position: absolute;
  top: -100%;
  background: ${uiColors('lightGray')};
  border: 1px solid ${uiColors('gray')};
  text-align: center;
  display: none;
  width: auto;
  font-size: 15px;
  line-height: 21px;
`;

export default TooltipInfo;
