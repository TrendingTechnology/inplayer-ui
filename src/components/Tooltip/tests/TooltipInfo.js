import { generateSnapshots } from 'helpers';

import TooltipInfo from '../TooltipInfo';

generateSnapshots(TooltipInfo);
