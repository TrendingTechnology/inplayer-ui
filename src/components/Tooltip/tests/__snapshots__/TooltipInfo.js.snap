// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TooltipInfo renders correctly 1`] = `
.c0 {
  position: absolute;
  top: -100%;
  background: #f0f3f6;
  border: 1px solid #d7dde5;
  text-align: center;
  display: none;
  width: auto;
  font-size: 15px;
  line-height: 21px;
}

<div
  className="c0"
/>
`;
